lib/feedsme/index.js

```diff
@@ -316,12 +362,15 @@ Feedsme.prototype._triggerOneDep = async function _triggerOneDep({ publish, prom
  * @returns {Object} strategy and whether we should trigger
  * @api private
  */
-Feedsme.prototype._triggerStrategy = function _triggerStrategy({ releaseLine, rootPkg, pkg, env }) {
+Feedsme.prototype._triggerStrategy = function _triggerStrategy({ registryRootPkg, releaseLine, rootPkg, pkg, env }) {
   if (env !== 'dev' && releaseLine) return { strategy: 'release', trigger: true };
   if (!releaseLine) return { strategy: 'legacy', trigger: true };
 
   const { name, version: rootVersion } = rootPkg;
   const { dependencies } = pkg;
+  const rootPkgVersions = Object.keys(registryRootPkg.versions || {})
+    .sort((y, x) => semver.compare(x, y));
+  const latestRootPkgVersion = rootPkgVersions[0];
```

Comment:
This would be more obvious using `.find` imho:

``` js
const latestRootPkgVersion = Object.keys(registryRootPkg.versions || {})
  // Sort in descending order 
  .sort((x, y) => semver.compare(y, x))
  // Return latest version 
  .find(v => v);
```

Reply:
huh, i guess thats not so bad. Changed to this and used `rcompare` to make it more clear.